app/services/folio_client.rb

```diff
@@ -44,6 +44,8 @@ def inspect
   def login_by_sunetid(sunetid)
     response = get_json('/users', params: { query: CqlQuery.new(username: sunetid).to_query })
     response&.dig('users', 0)
+  rescue RuntimeError
+    nil
```

Comment:
what raises this?

Reply:
Good catch; I think maybe we can remove this now (or we'll discover a need later).